src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractSyncNamespaceCodeRefactoringProvider.MoveFileCodeAction.cs

```diff
@@ -38,12 +38,6 @@ public MoveFileCodeAction(State state, ImmutableArray<string> newFolders)
 
             internal override bool PerformFinalApplicabilityCheck => true;
```

Comment:
this should be false.  THere is no need to perform a final applicability check now :)